src/oneD/Flow1D.cpp

```diff
@@ -514,55 +514,53 @@ void Flow1D::evalContinuity(double* x, double* rsd, int* diag,
 {
     // The left boundary has the same form for all cases.
     if (jmin == 0) { // left boundary
-        rsd[index(c_offset_U,jmin)] = -(rho_u(x,jmin + 1) - rho_u(x,jmin))/m_dz[jmin]
-                                      -(density(jmin + 1)*V(x,jmin + 1)
-                                      + density(jmin)*V(x,jmin));
-
-        diag[index(c_offset_U,jmin)] = 0; // Algebraic constraint
+        rsd[index(c_offset_U, jmin)] = -(rho_u(x, jmin+1) - rho_u(x, jmin))/m_dz[jmin]
+                                      -(density(jmin+1)*V(x, jmin+1)
+                                      + density(jmin)*V(x, jmin));
+        diag[index(c_offset_U, jmin)] = 0; // Algebraic constraint
     }
 
     if (jmax == m_points - 1) { // right boundary
         if (m_usesLambda) { // axisymmetric flow
             rsd[index(c_offset_U, jmax)] = rho_u(x, jmax);
         } else { // right boundary (same for unstrained/free-flow)
-            rsd[index(c_offset_U, jmax)] = rho_u(x, jmax) - rho_u(x, jmax - 1);
+            rsd[index(c_offset_U, jmax)] = rho_u(x, jmax) - rho_u(x, jmax-1);
         }
         diag[index(c_offset_U, jmax)] = 0; // Algebraic constraint
     }
 
     // j0 and j1 are constrained to only interior points
     size_t j0 = std::max<size_t>(jmin, 1);
-    size_t j1 = std::min(jmax, m_points - 2);
+    size_t j1 = std::min(jmax, m_points-2);
     if (m_usesLambda) { // "axisymmetric-flow"
         for (size_t j = j0; j <= j1; j++) { // interior points
             // For "axisymmetric-flow", the continuity equation  propagates the
             // mass flow rate information to the left (j+1 -> j) from the value
             // specified at the right boundary. The lambda information propagates
             // in the opposite direction.
-            rsd[index(c_offset_U,j)] = -(rho_u(x,j+1) - rho_u(x,j))/m_dz[j]
-                                       -(density(j+1)*V(x,j+1) + density(j)*V(x,j));
-
+            rsd[index(c_offset_U, j)] = -(rho_u(x, j+1) - rho_u(x, j))/m_dz[j]
+                                       -(density(j+1)*V(x, j+1) + density(j)*V(x, j));
```

Comment:
A minor nitpick, but the second line should be indented by one more space to align it with the first.